_posts/2018-05-25-ecs-calculator.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Cost efficiency while using the containers
+banner: /assets/posts/2018-05-25-ecs-calculator/ecs-banner.png
+author:
+  - veranikaisakova
+  - hadoan
+---
+
+In the cloud, containers provide a containerised environment enabling your code to be built, shipped and run anywhere. This can be simply done by just running your code without setting up your operating system.
+
+Coming with such benefit, AWS Elastic Container Services (ECS) is a a highly scalable, fast, container management service that makes easy to run your containerised code and applications across a managed cluster of EC2 instances. You will need to optimally scale and fit ECS containers in an EC2 instance for the efficient usage of your resources in the cloud. This means you should ensure that resources such as CPU and Memory units are enough and efficiently reserved for your largest container to scale out.
```

Comment:
`This means you should ensure that resources such as CPU and Memory units are enough for your largest container to scale out and efficiently reserved so that your containers can fill the instances in your deployment.`